### DISCUSSION TITLE
.net  + seq

### DISCUSSION THREAD

**Discussion by 861191244** [datalust/seq-tickets#1626] [Help]
.net  core  3.1 webapi application  deploy  in  centos  , the logs does not appear  in the  Seq Management UI  Events  ,but  application  
can be appear  where  app deploy   in  windows server 2012  ,the  centos firewall  is closed ,   server  all ports  also is opend , and  
 localhost(windows 10)  excute   dotnet  xxxx.dll   start  the application  can be appear   in the  Seq Management UI  Events (centos) , Why is it happen,how to solve? thanks

**Comment by nblumhardt** [comment]
Hi! Thanks for getting in touch.

There are some troubleshooting steps for this scenario at:

https://docs.datalust.co/docs/ingestion-troubleshooting

Unless you find that there are entries in the ingestion log (Data > Ingestion), it's most likely that the app running on Centos can't resolve the hostname of the Seq server, or can't reach it over the network.

If you spot any clues that we can dig into please let us know.

Nick
